qiita_pet/support_files/doc/source/admin/portals.rst

```diff
@@ -0,0 +1,107 @@
+.. _portals:
+
+.. index:: portals
+
+User Portals
+============
+User portals allow a user to cordon off specific studies and analyses into their
```

Comment:
This has not been fixed, still shows in bold:

<img width="741" alt="screen shot 2015-07-05 at 4 30 19 pm" src="https://cloud.githubusercontent.com/assets/2501478/8533642/398e7acc-23eb-11e5-9099-2d3637637f63.png">